tests/longhaul/base_block_test.go

```diff
@@ -155,6 +161,13 @@ func (o BaseLoadTestOperations) SetUp() {
 		isRookInstalled, err := o.installer.InstallRookOnK8sWithHostPathAndDevices(o.namespace, "bluestore", "/temp/rookBackup", false, true, 3)
 		require.NoError(o.T(), err)
 		require.True(o.T(), isRookInstalled)
+
+	}
+
+	// Enable chaos monkey if enable_chaos flag is present
+	if o.installer.Env.EnableChaos {
+		c := NewChaosHelper(o.namespace, o.kh)
+		go c.Monkey()
```

Comment:
how does the chaos monkey know when to stop? Does the whole process terminate when the load test is done?